test/pages/register-a-beacon/more-details.test.tsx

```diff
@@ -1,26 +0,0 @@
-import { render, screen } from "@testing-library/react";
-import React from "react";
-import { FormJSON } from "../../../src/lib/form/formManager";
-import MoreDetails from "../../../src/pages/register-a-beacon/more-details";
-
-describe("MoreDetails page", () => {
-  const emptyMoreDetailsForm: FormJSON = {
```

Comment:
@zackads To update these tests to check that the page correctly renders based on the input that is passed to the component.

Do not need to duplicate tests within Cypress though.